scss/form/selects.scss

```diff
@@ -49,8 +49,7 @@
   $types:
     "success" map-get($success-colors, "normal") map-get($success-colors, "hover"),
     "warning" map-get($warning-colors, "normal") map-get($warning-colors, "hover"),
-    "error" map-get($error-colors, "normal") map-get($error-colors, "hover"),
-    "dark" map-get($default-colors, "normal") map-get($default-colors, "hover");
```

Comment:
Not sure if this is intentional but by removing this line we lose the white border, like this:

![image](https://user-images.githubusercontent.com/22016005/71385268-23585d00-25c5-11ea-88d2-49627631accd.png)

I think the bug is that we need to add it first, rather than last. If you can try to add this in line 50 so the style remains the same:

![image](https://user-images.githubusercontent.com/22016005/71385295-4edb4780-25c5-11ea-907a-275abebf2f9b.png)

Reply:
My own ignorance in not checking the dark container, sorry about that.